sdk/documentintelligence/azure-ai-documentintelligence/samples/sample_analyze_receipts_from_url.py

```diff
@@ -0,0 +1,120 @@
+# coding: utf-8
+
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: sample_analyze_receipts_from_url.py
+
+DESCRIPTION:
+    This sample demonstrates how to analyze and extract common fields from a receipt URL,
+    using a pre-trained receipt model.
+
+    See fields found on a receipt here:
+    https://aka.ms/azsdk/formrecognizer/receiptfieldschema
```

Comment:
These links should really be updated to say document intelligence and not form recognizer anymore. Wont block on this now though